component-library/src/app/shell/shell.component.html

```diff
@@ -2,7 +2,9 @@
   <div class="header">
     <ircc-cl-lib-header [id]="'header'"></ircc-cl-lib-header>
     <div class="container">
-      <h1 class="emphasis">{{ 'Overview.SubHeading' | translate }}</h1>
+      <h2 [ngClass]="{ emphasis: true, h4: mobile }">
```

Comment:
I think it's better to use `strong` which has `font-weight: 700;` instead of `emphasis`(`font-weight: 600;`)